skr/pkg/factory/factory.go

```diff
@@ -10,13 +13,12 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/source"
-	"strings"
-	"time"
 )
 
 func InformerFactoryWithLabel(client dynamic.Interface, mgr ctrl.Manager, label, value string) (dynamicinformer.DynamicSharedInformerFactory, error) {
 	// Create informerFactory for each configured label
-	informerFactory := dynamicinformer.NewFilteredDynamicSharedInformerFactory(client, time.Minute*30, "", func(options *metav1.ListOptions) {
+	defaultResync := time.Minute * 30
```

Comment:
not sure how lint issues passed here. Didn't it report of a magic number error?

Reply:
If you just initialise it before using it, then the error disappears